packages/radix-vue/src/TagsInput/TagsInputRoot.vue

```diff
@@ -174,8 +174,10 @@ provideTagsInputRootContext({
       case 'ArrowLeft': {
         const isArrowRight = (event.key === 'ArrowRight' && dir.value === 'ltr') || (event.key === 'ArrowLeft' && dir.value === 'rtl')
         const isArrowLeft = !isArrowRight
-        // only focus on tags when cursor is at the first position
-        if (target.selectionStart !== 0 || target.selectionEnd !== 0)
+
+        // prevent tag focus when there's something in the input
+        // the user must clear the input if he needs to clear already added tags
+        if (target.value.length > 0)
```

Comment:
This is not the behavior we want. We want to allow user to focus on tags even when there's input value.

Reply:
I understand that's the behavior you intended. However, it's hijacking the default behavior of a text input. Its purpose is to allow a user to enter a new value or perform a search, therefore it is reasonable to expect it to behave like a normal text input.

From a user's perspective, typing indicates the intention of adding or searching. 
Therefore while he is typing, he should be able to select it as he normally would. That intention should be prioritized over the intention of focusing tags that were already added.

I believe you should reconsider this behavior. Otherwise feel free to close the PR.

Many thanks